AUTHORS

```diff
@@ -1099,4 +1099,32 @@ Joannah Nanjekye <joannah.nanjekye@ibm.com>
 Ayush Kumar <ayushk7102@gmail.com>
 Kshitij <kshitijparwani.mat18@itbhu.ac.in>
 Daniel Hyams <dhyams@gmail.com>
+alijosephine <alijosephine@gmail.com>
 Matthias Köppe <mkoeppe@math.ucdavis.edu>
+mohajain <mohajain99@gmail.com>
+Anibal M. Medina-Mardones <ammedmar@gmail.com>
+Travis Ens <ens.travis@gmail.com>
+Evgenia Karunus <lakesare@gmail.com>
+RDxR10 <risirajdey@gmail.com>
```

Comment:
@RDxR10 would you like this to show your real name?

Reply:
I would prefer "Risiraj Dey". Thanks

Reply:
Thanks. I've change that.